test/client_test/wabur.conf

```diff
@@ -0,0 +1,35 @@
+# wabur.conf
+
+# Configuration file for the Pure Ruby WAB Runner.
+
+# The directory to store the runner data (relative to source directory).
+store.dir = client_test/data
+
+# URL path prefix for the data records.
+path_prefix = /v1
+
+# Field to expect the type, class, or kind of record in the stored JSON.
+type_key = kind
+
+# HTTP site directory.
+http.dir =  site
+
+# HTTP port to listen on.
+http.port = 6373
+
+# Logging verbosity. Can be ERROR, WARN, INFO, or DEBUG.
+verbosity = WARN
+
+# If export_proxy is true the wab default index.html, CSS, JavaScript, and
+# fonts are loaded from the gem's export directory is they are not found in
+# the site directory.
+export_proxy = true
+
+# The number of spaces to indent the JSON written to disk by the model and the
+# JSON sent as a response to View requests.
+indent = 0
+
+# Handlers for each type of record and for the UI shich are Ruby generated UI
+# configuration records.
+handler.0.type = 
```

Comment:
empty declaration?

Reply:
yes, that is the wild card, catch everything notation.